.github/workflows/test.yml

```diff
@@ -0,0 +1,140 @@
+name: KBase Workspace tests
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+  push:
+    # run workflow when merging to main or develop
+    branches:
+      - main
+      - master
+      - develop
+
+# TODO find out what minio ver we're using in CI / appdev / prod and use that
+# TODO split tests into mongo related & non mongo related. Run the former once.
+jobs:
+
+  auth2_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - java: '8'
+            mongo: 'mongodb-linux-x86_64-3.6.23'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'false'
+            ant_test: 'test_quick_coverage'
+          - java: '11'
+            mongo: 'mongodb-linux-x86_64-3.6.23'
+            minio: '2019-05-23T00-29-34Z'
+            wired_tiger: 'true'
+            ant_test: 'test_quick_coverage'
```

Comment:
unlike the Travis tests, you just want to run java 8 / wired_tiger: false, and java 11 / wired_tiger: true; correct?

Reply:
yeah, the only reason I did it that way before was because I don't think there was a way to matrix in the java versions. Now you can, so the number of tests can be halved.

*Maybe* there's some crazy interaction between the Mongo storage engine and the java version, but that seems way too unlikely to double our test compute just in case